homeassistant/components/blink/camera.py

```diff
@@ -94,7 +104,6 @@ async def trigger_camera(self) -> None:
         """Trigger camera to take a snapshot."""
         with contextlib.suppress(asyncio.TimeoutError):
             await self._camera.snap_picture()
-        self.async_write_ha_state()
```

Comment:
Was this a mistake from before?

Reply:
I may have accidentally removed this here. I think we need it, as it updates the picture in HA after the update (right?)

Reply:
I never worked with the camera entity so I am not sure here

Reply:
I think this is broken here, after taking a snap (telling blink to do it anyway), we need to call the update function to download the new snap.  This explains why it takes "forever" to get the updated photos.  
For this PR, I think we should leave it as is, and I'll work out the issues and submit a new PR for in a bit.